from .dns_record_base_exception import DnsRecordBaseException


class AddError(DnsRecordBaseException):
    pass
